sklearn/decomposition/factor_analysis.py

```diff
@@ -88,6 +88,10 @@ class FactorAnalysis(BaseEstimator, TransformerMixin):
         Number of iterations for the power method. 3 by default. Only used
         if ``svd_method`` equals 'randomized'
 
+    rotation : None | 'varimax' | 'quartimax'
+        If not None, apply the indicated rotation. Currently, varimax and
```

Comment:
I suspect references for varimax and quartimax should be cited in the docstring, and cross-referenced here.